twindb_backup/destination/s3.py

```diff
@@ -196,6 +197,8 @@ def save(self, handler, name):
         except S3DestinationError as err:
             LOG.error('S3 upload failed: %s', err)
             raise err
+        except MySQLSourceError:
+            return False
```

Comment:
Why there is such inconsistent behavior? `S3DestinationError` is re-raised while `MySQLSourceError ` returns False